controller/ReferralController.js

```diff
@@ -1,6 +1,21 @@
 const catchAsyncError = require("../middleware/catchAsyncError");
 const { Referral } = require("../services");
 
+/**
+ * @description Create a referral short-form link
+ * @route POST v1/referral/link
+ * @access PUBLIC
+ */
+exports.createReferralLink = catchAsyncError(async (req, res) => {
```

Comment:
@Chiemezuo, similarly why is this showing as a new addition. Did you create a branch off `develop` to make these changes?

Reply:
It's a mistake with the PR itself. I'll fix and re-open.